docs/training/2025/webinar-series-2025Q2.md

```diff
@@ -86,6 +86,13 @@ and simplify your scientific workflows.
 #### Materials
 
 - [slides (PDF)](EESSI-webinars-MayJune-2025-001-Introduction-to-EESSI-20250505.pdf)
+- [recording (YouTube)]()
```

Comment:
there should be the link right?
`[recording (YouTube)](https://www.youtube.com/watch?v=FvVbzKLn-C8)`